README.md

```diff
@@ -90,6 +94,11 @@ For development you can also watch source files for changes and automatically li
 
     make watch
 
+## Web server
+
+Web server can run through the command line:
```

Comment:
@SteveMarshall I think it would be better if you have that part as well because it will help us when we will need to render test in a browser by viewing `SpecRunner.html`